src/CMakeLists.txt

```diff
@@ -57,6 +60,7 @@ set(freerct_SRCS ${freerct_SRCS}
     "${CMAKE_SOURCE_DIR}/src/generated/coasters_strings.h"
     "${CMAKE_SOURCE_DIR}/src/generated/gui_strings.h"
     "${CMAKE_SOURCE_DIR}/src/generated/shops_strings.h"
+    "${CMAKE_SOURCE_DIR}/src/generated/comp_info.h"
```

Comment:
I have a feeling it might be worth merging rev and comp_info together - we hardly need 2 different tiny files for compile-time information